src/components/ErrorBoundary/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as logging from '../../Logging';
+
+class ErrorBoundary extends React.Component {
+  componentDidCatch(error, info) { // eslint-disable-line no-unused-vars
```

Comment:
If you don't need the `info` variable, you don't need to provide it as an argument.